react-hyf/react-hyf-client/src/components/Navbar.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import Home from './Home';
+import 'bootstrap/dist/css/bootstrap.css';
+import { Component } from 'react';
+import Week1 from './Week1';
+import Week2 from './Week2';
+import Week3 from './Week3';
+
+import { BrowserRouter as Router, Switch, Link, Route } from 'react-router-dom';
+
+import {
+  Collapse,
+  Navbar,
+  NavbarToggler,
+  NavbarBrand,
+  Nav,
+  NavItem,
+  NavLink,
+  Container,
+} from 'reactstrap';
+class AppNavbar extends Component {
+  state = {
+    isOpen: false,
+  };
+  toggle = () => {
+    this.setState({
+      isOpen: !this.state.isOpen,
+    });
+  };
+  render() {
+    return (
+      <React.Fragment>
+        <Navbar className="navbar navbar-expand-lg navbar-light bg-primary ">
+          <Container>
+            <NavbarBrand>React homework</NavbarBrand>
+            <NavbarToggler onClick={this.toggle} />
+            <Collapse isOpen={this.state.isOpen} navbar>
```

Comment:
These two `NavbarToggler` & `Collabse` components aren't in use as they've expected to be.